pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -990,3 +979,135 @@ func TestHighProirotyFlushUnschedulableQLeftover(t *testing.T) {
 		t.Errorf("Expected: %v after Pop, but got: %v", medPriorityPod.Name, p.Name)
 	}
 }
+
+// TestHighProirotyPodInfo tests the operations related to podInfo.
+func TestHighProirotyPodInfo(t *testing.T) {
```

Comment:
I would rename this to something like `TestPodTimestamp` or something like that. In this test we don't care much about priority. So, having "HighPriority" in the name of the test is not necessary.

Reply:
Done thank you